packages/frontend/app/routes/tenants.$tenantId.tsx

```diff
@@ -237,43 +246,97 @@ export default function ViewTenantPage() {
 }
 
 export async function action({ request }: ActionFunctionArgs) {
-  const errors: {
-    fieldErrors: ZodFieldErrors<typeof createTenantSchema>
-    message: string[]
+  const actionResponse: {
+    errors: {
+      general: {
+        fieldErrors: ZodFieldErrors<typeof updateTenantGeneralSchema>
+        message: string[]
+      }
+      ip: {
```

Comment:
nit: the idp schema is being dereferenced as "ip" here

Reply:
fixed [f95153b](https://github.com/interledger/rafiki/pull/3328/commits/f95153b128ff1a5b6548447254fe936fee4aeb63)